magpie/adapter/__init__.py

```diff
@@ -68,7 +67,7 @@ def debug_cookie_identify(request):
             LOGGER.debug("Cookie is expired")
 
         # Could raise useful exception explaining why unauthenticated_userid is None
-        request._get_authentication_policy().cookie.identify(request)
+        request._get_authentication_policy().cookie.identify(request)  # noqa: W0212
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _get_authentication_policy of a client class](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)